utils/gen_onnx_mlir.py

```diff
@@ -485,7 +491,7 @@ def onnx_attr_type_to_mlir_attr_type(t):
         mlir_attr_type = 'StrAttr'
     elif onnx_attr_type == "strings":
         mlir_attr_type = 'StrArrayAttr'
-    elif onnx_attr_type == 'type':
+    elif onnx_attr_type in {'type', 'type_proto'}:
```

Comment:
Can you just comment on what this new `type_proto` is for us not familiar with the onnx types, tx

Reply:
added the comment:
```
        # 'type' is the attribute type used in special_attr_types,
        # 'type_proto' is Optional op's type attribute's type
```